# AssertionError with error_message.
try:
 x = 1
 y = 0
 assert y != 0
 print(x / y)
except:
    print("Uzur Invalid Operation")

# Errorni ko'rmoqchi bolsangiz try va exceptni o'chiring !
